document-store/src/integrationTest/java/org/hypertrace/core/documentstore/DocStoreQueryV1Test.java

```diff
@@ -349,6 +350,178 @@ public void testFindWithNestedFields(String dataStoreName) throws IOException {
     testCountApi(dataStoreName, query, "query/filter_on_nested_fields_response.json");
   }
 
+  @ParameterizedTest
+  @MethodSource("databaseContextPostgres")
+  void testAggregateWithId(String dataStoreName) throws IOException {
```

Comment:
Could you re-use the existing `collection_data.json` to put the additional fields instead of creating a separate document for each test?